bin/utest/exec.c

```diff
@@ -0,0 +1,54 @@
+#include "utest.h"
+
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/wait.h>
+#include <assert.h>
+#include <stdlib.h>
+
+int test_execve() {
```

Comment:
The signature of this function is different in the header file `int test_execve(void)`.
In C `foo(void)` is different than `foo()`. Second variant could take any number of parameters of unknown types.

The same comment applies to `test_execv`.